gamemodes/terrortown/gamemode/shared/sh_door.lua

```diff
@@ -232,7 +232,7 @@ if SERVER then
 	function entmeta:SetDoorCanTouchOpen(state, surpressPair)
 		door.SetPlayerCanTouch(self, state)
 
-		self:SetNWBool("ttt2_door_player_touch", PlayerCanTouchDoor(self))
+		self:SetNWBool("ttt2_door_player_touch", door.PlayerCanTouch(self))
```

Comment:
I have to check the code, but I'm pretty sure that change is wrong

Reply:
couldnt find anything better, but PLayerCanTouchDoor is not a function anywhere

Reply:
I used git blame a bit and I guess you're right. At some point we moved these local functions into global functions.

old:
![image](https://github.com/TTT-2/TTT2/assets/13639408/4b50281e-ac81-4ccd-baee-460cc85b9f88)

new:
![image](https://github.com/TTT-2/TTT2/assets/13639408/800e58ce-938c-447b-96ee-caf77eabd7cb)

Reply:
Good to know, I tested a lot and those functions are never called. They are not even referenced inside TTT2, might they have become useless?

Reply:
such as `door.PlayerCanTouch`? These are used for targetID by setting the NW bool on the server. They are also an external API that can be used by addons - if they want to. So I don't see why we shouldn't keep them.

Reply:
![grafik](https://github.com/TTT-2/TTT2/assets/72046466/799f8a99-9b80-44b2-8a91-12c5cc84ea56)
These 3 here are not called anywhere, might they be deprecated?

Reply:
I meant entmeta:SetDoorCanTouchOpen and CanUseOpen and AutoCloses

Reply:
I don't think I intended on using them in TTT2, I added them as part of the API so addons can use this on entities. I don't know if any addon uses this, but I wouldn't say that it is deprecated.

I know for a fact that i use `entmeta:ToggleDoor` in one of my addons for example. As well as `entmeta:LockDoor`. I therefore exposed all settings to the API, instead of just those I needed at the time.

Reply:
> I don't think I intended on using them in TTT2, I added them as part of the API so addons can use this on entities. I don't know if any addon uses this, but I wouldn't say that it is deprecated.
> 
> I know for a fact that i use `entmeta:ToggleDoor` in one of my addons for example. As well as `entmeta:LockDoor`. I therefore exposed all settings to the API, instead of just those I needed at the time.

Yeah those I tested. Okay well, thanks for the input ❤️

Reply:
> I meant entmeta:SetDoorCanTouchOpen and CanUseOpen and AutoCloses

It is probably not uses right now, yes. But because this is a general API for doors, I don't see how it is deprecated.

One could make an addon that is an admin tools to setup doors for example